handlers.js

```diff
@@ -70,13 +74,36 @@ export function makeRows({tests, columns}) {
     row.cells[columnIndex] = current;
     return rows;
   }, []);
+  return _rows.map(({id, cells}) => {
+    for(const colName of notImplemented) {
+      const columnIndex = columnIds.indexOf(colName);
+        cells[columnIndex] = {
+          cell: {
+            rowId: id,
+            colId: colName
+          },
+          state: 'notImplemented'
+        }
```

Comment:
```suggestion
      cells[columnIndex] = {
        cell: {
          rowId: id,
          colId: colName
        },
        state: 'notImplemented'
      };
```

Reply:
should be addressed here: https://github.com/digitalbazaar/mocha-w3c-interop-reporter/pull/7/commits/4585e53dd271bcf777588d558747dd6eb3366102